com.albiononline.AlbionOnline.json

```diff
@@ -0,0 +1,82 @@
+{
+    "app-id": "com.albiononline.AlbionOnline",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "albion-online",
+    "tags": ["proprietary"],
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
```

Comment:
Realized that my emacs was not well configured and there was a mix of tabs and spaces. Maybe the patch looked misindented because of that.